pyproject.toml

```diff
@@ -115,6 +116,7 @@ test = [
     "pytest",     # Required for unit tests
     "pytest-cov", # Reports on test coverage
     "pixelmatch",
+    "xarray-datatree", # Currently only used in unit tests
```

Comment:
@jsolbrig @evrose54 @AJThorpe  I added xarray-datatree to optional "test" pacakges here, since it is neede for the pytests.